app/models/language_skill.rb

```diff
@@ -5,6 +5,8 @@ class LanguageSkill < ApplicationRecord
 
   def language_name
     return '' if language.blank?
+    return 'Dari' if language == 'DR'
+    return 'Farsi' if language == 'FS'
```

Comment:
It would be great if we could add custom languages to I18nData so we don't have to duplicate it here, but the gem doesn't seem to support that.